mkdocs_rss_plugin/plugin.py

```diff
@@ -115,14 +115,20 @@ def on_config(self, config: MkDocsConfig) -> MkDocsConfig:
             "author": config.site_author or None,
             "buildDate": formatdate(get_build_timestamp()),
             "copyright": config.copyright,
-            "description": config.site_description,
+            "description": (
+                self.config["description"]
+                if self.config["description"]
+                else config["site_description"]
+            ),
```

Comment:
Since Mkdocs 1.4, most of config values are attributes not dict key/values no? Can you check that we fit the modern way to access Makdocs config object here please?